<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datasetDlg</class>
 <widget class="QDialog" name="datasetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Dataset</string>
  </property>
  <widget class="QLabel" name="newDataset">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>New Dataset</string>
   </property>
  </widget>
  <widget class="QLabel" name="specification">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dataset Specification</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="datablockSize">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Datablock Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="datablockFormat">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Datablock Format:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="datablockSizeEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Minimum 100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="datablockFormatEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ex: LAZ</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Adquisition:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>DD/MM/YYYY</string>
   </property>
  </widget>
  <widget class="QLabel" name="boundingBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bounding Box</string>
   </property>
  </widget>
  <widget class="QLabel" name="south">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>South-West Bottom</string>
   </property>
  </widget>
  <widget class="QLabel" name="north">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>North-East Top</string>
   </property>
  </widget>
  <widget class="QLabel" name="southEasting">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Easting:</string>
   </property>
  </widget>
  <widget class="QLabel" name="southNorthing">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Northing:</string>
   </property>
  </widget>
  <widget class="QLabel" name="southHeight">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="southZone">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zone:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="southEastingEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ex: 826083</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="southNorthingEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ex: 1158000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="southHeightEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ex: 0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="southZoneEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>UTM Zone - Ex:16N</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="northNorthingEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ex: 1159506</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="northEastingEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Ex: 829575</string>
   </property>
  </widget>
  <widget class="QLabel" name="northNorthing">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Northing:</string>
   </property>
  </widget>
  <widget class="QLabel" name="northEasting">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Easting:</string>
   </property>
  </widget>
  <widget class="QLabel" name="northHeight">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="northHeightEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ex: 0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="northZoneEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>UTM Zone - Ex:17S</string>
   </property>
  </widget>
  <widget class="QLabel" name="northZone">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zone:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>468</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>datablockSizeEdit</tabstop>
  <tabstop>datablockFormatEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>southEastingEdit</tabstop>
  <tabstop>southNorthingEdit</tabstop>
  <tabstop>southHeightEdit</tabstop>
  <tabstop>southZoneEdit</tabstop>
  <tabstop>northEastingEdit</tabstop>
  <tabstop>northNorthingEdit</tabstop>
  <tabstop>northHeightEdit</tabstop>
  <tabstop>northZoneEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>datasetDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>620</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>datasetDlg</receiver>
   <slot>finish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>886</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>finish()</slot>
 </slots>
</ui>
